app/phone/build.gradle.kts

```diff
@@ -110,6 +110,7 @@ dependencies {
     implementation(libs.androidx.work)
     implementation(libs.glide)
     implementation(libs.hilt.android)
+    implementation("com.google.android.gms:play-services-cast-framework:21.3.0")
```

Comment:
Can move this into version catalogs in [libs.versions.toml](https://github.com/deHank/findroid/blob/chromecastMain/gradle/libs.versions.toml)